docs/faq.md

```diff
@@ -0,0 +1,81 @@
+---
+uid: faq
+title: Frequently Asked Questions
+---
+
+# Frequently Asked Questions
+
+### I've updated from an old version to the latest version and my project won't build!
+Please read the latest [migration article](xref:migration_3_4) to see a list of changes, as major version releases will usually have several breaking changes.
+
+### Code I copied from an article isn't compiling or working as expected. Why?
+*Please use the code snippets as a reference; don't blindly copy-paste code!*
+
+The snippets of code in the articles are meant to serve as examples to help you understand how to use a part of the library.<br/>
+Although most will compile and work at the time of writing, changes to the library over time can make some snippets obsolete.<br/>
+Many issues can be resolved with Intellisense by searching for similarly named methods and verifying method parameters.
+
+### I'm targeting Mono and have exceptions, crashes, or other problems.
+As mentioned in the [preamble](xref:preamble), the Mono runtime is inherently unstable and has numerous flaws.<br/>
+Because of this we do not support Mono in any way, nor will we support any other projects which use it.
+
+Instead, we recommend using the latest LTS release of [.NET Core](https://dotnet.microsoft.com/download). 
+
+### Connecting to a voice channel with VoiceNext will either hang or throw an exception.
+To troubleshoot, please ensure that:
+
+* You are using the latest version of DSharpPlus.
+* You have properly enabled VoiceNext with your instance of `DiscordClient`.
+* You have [opus and libsodium](xref:voicenext_prerequisites) available in your target environment.
+
```

Comment:
Also worth mentioning that attempting to do so in an event handler will cause a deadlock